lua/wire/client/texteditor.lua

```diff
@@ -3656,33 +3670,9 @@ do
 				tokenname = "comment"
 			elseif (self.character == "#") then
 				self:NextCharacter()
-				
-				if self:NextPattern("include +<") then
-					
-					cols[#cols + 1] = {self.tokendata:sub(1,-2), colors["pmacro"]}
-					
-					self.tokendata = "<"
-					if self:NextPattern("^[a-zA-Z0-9_/\\]+%.txt>") then
-						tokenname = "filename"
-					else
-						self:NextPattern(".*$")
-						tokenname = "normal"
-					end
-				elseif self:NextPattern("include +\"") then
```

Comment:
Is it just me or does this change remove matching `#include "foo"`? Is there any reason why?